.github/workflows/ci.yaml

```diff
@@ -253,17 +257,96 @@ jobs:
             --build-arg DISTRO_NAME \
             --build-arg DISTRO_VER \
             -t condaforge/$DOCKERIMAGE:$DOCKERTAG \
+            -t ${{ env.REGISTRY_DH }}/$DOCKERIMAGE:$DOCKERTAG \
+            -t ${{ env.REGISTRY_QUAY }}/$DOCKERIMAGE:$DOCKERTAG \
             -f ${DOCKERFILE:-${DOCKERIMAGE}}/Dockerfile \
             --no-cache --squash .
 
       - name: Run image
         run: |
           ./.circleci/run_docker_build.sh
 
-      - name: Deploy
+      - name: Deploy images
         if: github.ref == 'refs/heads/main' && github.repository == 'conda-forge/docker-images'
         env:
+          CFD_QUAY_USER: conda_forge_daemon
           CFD_QUAY_PASSWORD: ${{ secrets.CFD_QUAY_PASSWORD }}
+          DH_USER: condaforgebot
           DH_PASSWORD: ${{ secrets.DH_PASSWORD }}
         run: |
           ./scripts/deploy
+  
+  build-manifest:
+    needs: [build]
+    if: github.ref == 'refs/heads/main' && github.repository == 'conda-forge/docker-images'
+    name: ${{ matrix.docker-registry }}/${{ matrix.cfg.DOCKER_MANIFEST }}:${{ matrix.cfg.DOCKER_TAG }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        docker-registry: [docker.io/condaforge, quay.io/condaforge]
```

Comment:
It is not possible to use the env vars here (`REGISTRY_DH` and `REGISTRY_QUAY`) :-/

Reply:
Unfortunately this also does not work!
commit: https://github.com/martin-g/docker-images/pull/4/commits/426b61256ffcc2b15165cd7f57ea60d0ff230cf2
error: https://github.com/martin-g/docker-images/actions/runs/7474578018 
Fails with: 
```
Invalid workflow file: .github/workflows/ci.yaml#L291
You have an error in your yaml syntax on line 291
```